CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# [2.13.0](https://github.com/wix/wix-ui-tpa/compare/v2.12.1...v2.13.0) (2020-12-14)
+
+
+### Bug Fixes
+
+* **Toast:** Fix close button alignment ([1c285b7](https://github.com/wix/wix-ui-tpa/commit/1c285b75529cbc14b2198b8c10bf3d901ad01a3f))
+* **Dropdown:** Pass initialSelectedId to CoreDropdown when it updates ([ef3bff9](https://github.com/wix/wix-ui-tpa/commit/ef3bff981a360f2f87d48128010b3fc0819d3c74))
+
+
+### Features
+
+* **Button:** Use smart contrast on hover ([7eecbb0](https://github.com/wix/wix-ui-tpa/commit/7eecbb024534bab827650884a99e782e1c4721a5))
+* **RadioButton, Checkbox:** Add disabled mode state for theme box and some bug fixes. ([7e81f40](https://github.com/wix/wix-ui-tpa/commit/7e81f40bbe8311df5610c3fea4ffe4550c156d85))
+* **DatePickerInput:** - Add for working in progress ([2847526](https://github.com/wix/wix-ui-tpa/commit/2847526b051f126ec0afab3142f883e2c43a7a72))
+
+### Reverts
+
+* "fix(Button): Fix prefix and suffix alignments" ([8a18c7b](https://github.com/wix/wix-ui-tpa/commit/8a18c7b6edb547d7849db63cde2fd8bf98e68728))
```

Comment:
Maybe it should be under "Bug Fixes"?

Reply:
🤔  I think it's more accurate to use "revert" here, since this is a revert of a bug fix.
So it's a fix of a bug fix 😄 .

Reply:
I used the "revert" commit message as defined here: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#revert-commits